packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -255,7 +285,45 @@ export class Pipeline extends cdk.Construct implements cpapi.IPipeline {
 
   // ignore unused private method (it's actually used in Stage)
   // @ts-ignore
-  private _generateOutputArtifactName(stage: cpapi.IStage, action: cpapi.Action): string {
+  private _attachAction(stage: Stage, action: actions.Action): void {
+    // handle cross-region Actions here
+    if (action.region) {
+      // get the region the Pipeline itself is in
+      const pipelineStack = cdk.Stack.find(this);
+      const pipelineRegion = pipelineStack.requireRegion(
+        "You need to specify an explicit region when using CodePipeline's cross-region support");
+      const pipelineAccount = pipelineStack.requireAccountId(
```

Comment:
Technically you only need the account ID if you are creating the replication buckets yourself, correct? 
I am also wondering if we can just use the default account ID if an account ID is not explicitly defined.

Reply:
> Technically you only need the account ID if you are creating the replication buckets yourself, correct?

Good call out. Fixed.

> I am also wondering if we can just use the default account ID if an account ID is not explicitly defined.

We cannot - that would make the hash calculation always return the same value for every customer using this functionality, which means their Bucket names would clash.